cli_079_rw0_universal_thermal_climate_index/cli_079_rw0_universal_thermal_climate_index_processing.py

```diff
@@ -0,0 +1,206 @@
+import os
+import sys
+import cdsapi
+from zipfile import ZipFile
+from dotenv import load_dotenv
+load_dotenv()
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')), 'utils')
+if utils_path not in sys.path:
+   sys.path.append(utils_path)
+import util_files
+import netCDF4
+import numpy as np
+import rasterio
+import fnmatch
+import logging
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of asset on GEE where you want to upload data
+# this should be an asset name that is not currently in use
+dataset_name = 'cli_079_rw0_universal_thermal_climate_index'
+
+logger.info('Executing script for dataset: ' + dataset_name)
+
+'''
+Download data
+'''
+
+# create a new sub-directory within your specified dir called 'data'
+# within this directory, create files to store raw and processed data
+data_dir = util_files.prep_dirs(dataset_name)
+
+# declare years and months to download data for
+# NOTE: data is large -- files for 2021 + 2022 are 13G total and take a long time to download
```

Comment:
@chrowe @weiqi-tori  We should chat about how many months to download, as it takes a long time, is a lot of data, and has a bit of nuance to it.